lib/child_process.js

```diff
@@ -613,8 +596,37 @@ function normalizeSpawnArguments(file, args, options) {
 
 function spawn(file, args, options) {
   const child = new ChildProcess();
-
   options = normalizeSpawnArguments(file, args, options);
+
+  if (options.signal) {
+    const signal = options.signal;
+    // Validate signal, if present
+    validateAbortSignal(signal, 'options.signal');
+
+    // Do nothing and throw if already aborted
+    if (signal.aborted) {
+      onAbortListener();
+    } else {
+      const config = { once: true };
+      const remove = () => {
+        signal.removeEventListener('abort', onAbortListener, config);
```

Comment:
`removeEventListener` doesn't take an option object.

```suggestion
        signal.removeEventListener('abort', onAbortListener);
```

Reply:
Done in 026021a
Please have a look 🙂